.dockerignore

```diff
@@ -0,0 +1,4 @@
+# More info: https://docs.docker.com/engine/reference/builder/#dockerignore-file
```

Comment:
can we just reference `.gitignore` instead? see https://github.com/kubernetes-sigs/kueue/blob/main/.dockerignore